lms/djangoapps/teams/static/teams/templates/instructor-tools.underscore

```diff
@@ -0,0 +1,13 @@
+<div class="wrapper-msg">
+    <span class="left-floater">
```

Comment:
@efischer19 can we make this an <h3> so that logically follows the parent Heading "Edit Team"?